cairo1-run/src/cairo_run.rs

```diff
@@ -356,97 +362,112 @@ fn create_entry_code(
     // The builtins in the formatting expected by the runner.
     let (builtins, builtin_offset) =
         get_function_builtins(&signature.param_types, copy_to_output_builtin);
-    let mut ctx = casm! {};
+    let mut ctx = CasmBuilder::default();
```

Comment:
@orizi Can you add some documentation here?
Specially in the casm_extend! cases?

Reply:
do note this is *exactly* the same code as before - only where the code may somewhat be understandable by variable name.

add some additional doc.